runtime/slice.go

```diff
@@ -156,6 +156,58 @@ func initSliceType(map[string]*Object) {
 	SliceType.slots.Repr = &unaryOpSlot{sliceRepr}
 }
 
+func sliceIndex(f *Frame, o *Object) (i int64, raised *BaseException) {
```

Comment:
maxIntBig, minIntBig are 2147483647,-2147483648
While sys.maxsize, and sys.maxsize + 1 is 9223372036854775807, 9223372036854775808L
Is there a way to check it without using int64?